lib/offsite_payments/integrations/world_pay.rb

```diff
@@ -230,6 +230,12 @@ def custom_params
           return @custom_params ||= read_custom_params
         end
 
+        # Check if the request comes from IP range 195.35.90.0 – 195.35.91.255
+        def valid_sender?(ip)
+          return true if OffsitePayments.mode == :test 
+          !!(/^195\.35\.(9[0-1])\.([0-9]|[1-9][0-9]|1([0-9][0-9])|2([0-4][0-9]|5[0-5]))$/.match(ip))
```

Comment:
:+1: to that. I think you want `IPAddr.new("195.35.90.0/23")` if my subnet mask knowledge is still up to snuff